lib/ansible/modules/crypto/openssl_publickey.py

```diff
@@ -214,6 +221,10 @@ def generate(self, module):
 
                 if self.backup:
                     self.backup_file = module.backup_local(self.path)
+
+                if self.pubkey_comment is not None:
+                    publickey_content = publickey_content + (" " + self.pubkey_comment).encode('utf-8')
```

Comment:
I think this only makes sense for OpenSSH keys. For OpenSSL PEM keys, you don't want to append text in the same line directly after `----- END PUBLIC KEY -----`.